src/main/java/com/facebook/openwifirrm/mysql/DatabaseManager.java

```diff
@@ -482,6 +482,8 @@ public void addWifiScan(
 				stmt.setLong(4, entry.last_seen);
 				stmt.setInt(5, entry.signal);
 				stmt.setInt(6, entry.channel);
+				// TODO is this right? How do I test this?
```

Comment:
```suggestion
				// TODO : untested
```

Reply:
Will change in the new PR (since we're splitting this PR into two).